dds_cli/account_manager.py

```diff
@@ -77,17 +77,37 @@ def add_user(self, email, role, project, unit=None, no_mail=False):
         except simplejson.JSONDecodeError as err:
             raise dds_cli.exceptions.ApiResponseError(message=str(err))
 
+        errors = response_json.get("errors")
+        error_messages = dds_cli.utils.parse_project_errors(errors=errors)
+
         # Format response message
         if not response.ok:
             message = "Could not add user"
             if response.status_code == http.HTTPStatus.INTERNAL_SERVER_ERROR:
-                raise dds_cli.exceptions.ApiResponseError(message=f"{message}: {response.reason}")
+                raise dds_cli.exceptions.ApiResponseError(message=message)
+
+            message += ": " + response_json.get("message", "Unexpected error!")
+            show_warning = True
+            if error_messages:
+                message += f"\n{error_messages}"
+                show_warning = False
 
             raise dds_cli.exceptions.DDSCLIException(
-                message=f"{message}: {response_json.get('message', 'Unexpected error!')}"
+                message=message,
+                show_emojis=show_warning,
             )
 
-        dds_cli.utils.console.print(response_json.get("message", "User successfully added."))
+        message = response_json.get("message")
+        if message:
+            LOG.info(message)
+
+        if errors:
```

Comment:
Why are you not using the `error_messages` from line 82 here and instead parse them again?

Reply:
Yeah I'll have a look again.

Reply:
Fixed.